activemodel/lib/active_model/validations/numericality.rb

```diff
@@ -184,6 +186,9 @@ module HelperMethods
       #     validates_numericality_of :width, less_than: ->(person) { person.height }
       #     validates_numericality_of :width, greater_than: :minimum_weight
       #   end
+      #
+      # A check passes when a proc or a method call return nil.
```

Comment:
We should mention that a check passes unless `:equal_to`

Reply:
✔️